README.md

```diff
@@ -184,19 +184,19 @@ USAGE:
     simple [FLAGS] [OPTIONS]
 
 FLAGS:
-    -h, --help             Prints help information
-    -l, --list             Shows list of all possible Goose tasks and exits
-    -g, --log-level        Log level (-g, -gg, -ggg, etc.)
-        --manager          Enables manager mode
-        --no-hash-check    Ignore worker load test checksum
-        --no-stats         Don't print stats in the console
-        --only-summary     Only prints summary stats
-        --reset-stats      Resets statistics once hatching has been completed
-        --status-codes     Includes status code counts in console stats
-        --sticky-follow    User follows redirect of base_url with subsequent requests
-    -V, --version          Prints version information
-    -v, --verbose          Debug level (-v, -vv, -vvv, etc.)
-        --worker           Enables worker mode
+    -h, --help              Prints help information
+    -l, --list              Shows list of all possible Goose tasks and exits
+    -g, --log-level         Log level (-g, -gg, -ggg, etc.)
+        --manager           Enables manager mode
+        --no-hash-check     Ignore worker load test checksum
+        --no-reset-stats    Resets statistics once hatching has been completed
+        --no-stats          Don't print stats in the console
```

Comment:
s/stats/statistics

Reply:
That conflicts with `--stats-log-file`, `--stats-log-format`, and `--no-stats`. We have very consistently abbreviated "statistics" to stats, IMO to change this would be a larger followup issue.

Reply:
I meant in the description - it’s fine and wanted to be short for the argument. My RegExp would have  done the wrong thing though, so you win ;)

Reply:
While I've always preferred full names, the formatting of this help is auto-generated, and if the descriptions get too long it does weird wrapping. Expanding `stats` everywhere in the description may not be easily possible, we'll see in the followup.

Reply:
https://github.com/tag1consulting/goose/issues/131